shared/constants/form.ts

```diff
@@ -29,6 +29,7 @@ const FORM_SETTINGS_FIELDS = <const>[
   'submissionLimit',
   'title',
   'webhook',
+  'endPage',
```

Comment:
same here, don't think we should think about the endPage as part of the settings